hendrix/utils/__init__.py

```diff
@@ -17,7 +24,7 @@ def get_pid(options):
     """returns The default location of the pid file for process management"""
     namespace = options['settings'] if options['settings'] else options['wsgi']
     return '%s/%s_%s.pid' % (
-        HENDRIX_DIR, options['http_port'], namespace.replace('.', '_')
+        PID_DIR, options['http_port'], namespace.replace('.', '_')
```

Comment:
Why not cross-platform here (slashes)?
Like,
```
return os.path.join('{}', '{}_{}.pid').format(PID_DIR, options['http_port'], namespace.replace('.', '_')
```
(warning, I didn't test this one)

Reply:
:heavy_check_mark: